Specs/Number/Dutch/PercentModel.json

```diff
@@ -84,11 +78,10 @@
     "NotSupportedByDesign": "javascript,python,java"
   },
   {
-    "Input": "percentage van 30 procent",
```

Comment:
Isn't this a possible form? If so, we should tag all, right?

Reply:
I am not 100% sure. I translated this one initially, but changed it since `percentage van 30 procent` is double. It should be `percentage van 30` or `30 procent` in my opinion. I couldn't find any reference for this edge case. Should I add this one as NotSupported for now?

Reply:
I added it for now on our internal planner board, where we can review it with more native Dutch people.